src/runtime/flags.ts

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
Should this be in the runtime 'global'?

Reply:
I just tried moving it to Runtime, but it involved a bit too much plumbing. The test file creates a new Runtime object for every test, and that object isn't plumbed through to the Arc class which needs to access the flags. It doesn't seem worth jumping through all those hoops for what essentially boils down to a static variable in disguise...